crates/sparrow-runtime/src/lib.rs

```diff
@@ -40,6 +41,7 @@ pub use batch::*;
 pub use metadata::*;
 pub use read::*;
 use sparrow_api::kaskada::v1alpha::execute_request::Limits;
+pub use stores::*;
```

Comment:
You shouldn't need both `pub mod stores` and `pub use stores::*`. Generally prefer `pub mod` when you want to preserve the import path, if it gives context when used. For example, with `pub mod stores`, other files would import the `path()` function in object_stores_url like `sparrow-runtime::stores::ObjectStoreUrl::path(..)`. But if you use `pub use stores::*`, then someone could call that function with `sparrow-runtime::ObjectStoreUrl::path(..)`.